source/Main.hx

```diff
@@ -8,6 +8,6 @@ class Main extends Sprite
 	public function new()
 	{
 		super();
-		addChild(new FlxGame(0, 0, PlayState));
+		addChild(new FlxGame(1000, 500, PlayState));
```

Comment:
This does change the dimensions of what will always be displayed in the game window, but it doesn't actually resize created window to match. 

To make the window match the dimensions you picked here, you'll need to change a line in your `project.xml`:
```
	<window width="640" height="480" fps="60" background="#000000" hardware="true" vsync="false" />
```
should become
```
	<window width="1000" height="500" fps="60" background="#000000" hardware="true" vsync="false" />
```